elegant-emacs-common.el

```diff
@@ -142,12 +142,12 @@ background color that is barely perceptible."
 ;;; Mode line rendering
 ;;; -------------------------------------------------------------------
 ;;; This line below makes things a bit faster
-(set-fontset-font "fontset-default"  '(#x2600 . #x26ff) "Fira Code 16")
+;;; (set-fontset-font "fontset-default"  '(#x2600 . #x26ff) "Fira Code 16")
```

Comment:
Not sure how to replace this line which is not totally mandatory but make things faster. This is actually used to set the ↩︎ symbol using Fira Code. If this is not set, Emacs needs to look for a font where this symbol is defind and it tends to make things slow (on my machin at least).

Reply:
Yes that's what I figured but in the Melpa pull they say that it's not something a theme should do. This is a quick fix, once approved the fetchers will look for the files so we can do anything to them apart from renaming